src/qt/textctrl.cpp

```diff
@@ -21,10 +21,14 @@ class wxQtLineEdit : public wxQtEventSignalHandler< QLineEdit, wxTextCtrl >
 {
 public:
     wxQtLineEdit( wxWindow *parent, wxTextCtrl *handler );
-
+protected:
+    virtual void keyPressEvent(QKeyEvent *event) wxOVERRIDE;
 private:
     void textChanged(const QString &text);
     void returnPressed();
+#if QT_VERSION >= QT_VERSION_CHECK( 5, 12, 0)
```

Comment:
This condition is repeated several times and is really not very readable. Could you please define `QT_HAS_INPUT_REJECTED` (or something making even more sense if you can think of a better name) preprocessor symbol once, using this expression, and then test for it instead?